### DISCUSSION TITLE
BGP Neighbor shutdown

### DISCUSSION THREAD

**Discussion by ecrosswhite** [aristanetworks/avd#3096] [Q&A] [answered]
Hello, I am working to port our AVD 3.7 over to AVD 4.2 and one issue I'm facing is if I have shutdown a BGP neighbor within my config I get an error.

Error;
`ERROR! [DC2-LEAF-02]: 'Validation Error: vrf_blue[0].vrfs[0].bgp_peers[13]': Unexpected key(s) 'shutdown' found in dict.`

YML for reference;

>bgp_peers: 
> 10.1.1.73:
>   remote_as: 65000
>   description: DC1-DIST-1
>   nodes: [DC1-LEAF-BSL-01, DC1-LEAF-BSL-02]
>   send_community: all
>   route_map_out: RM_DIST
>   shutdown: true
> 10.1.1.74:
>   remote_as: 65000
>   description: DC1-DIST-1
>   nodes: [DC1-LEAF-BSL-01, DC1-LEAF-BSL-02]
>   send_community: all
>   route_map_out: RM_DIST
>   shutdown: true

I suspect with 3.7 it was using eos_cli_config_gen for the shutdown command, but with the upgrade it no longer uses this.  I don't see that shutdown is available with eos_designs.  Does anyone have a suggestion on how to keep the neighbor shutdown command?

Thanks

**Comment by ClausHolbechArista** [comment, marked as answer]
Hi @ecrosswhite, thank you for opening this discussion.

It seems like `shutdown` worked "by chance" in 3.x. I checked the [old docs](https://avd.sh/en/releases-v3.8.x/roles/eos_designs/doc/network-services.html#tenant-network-service-definitions) and since `shutdown` was not documented there, it was not carried over to the schema for 4.0.

We will create a fix to update the schema and release 4.3 including that fix later this week. Until then you can relax the schema enforcement by setting `avd_data_validation_mode: warning` and then I believe it will work. Please confirm.

Thanks
Claus

**Reply by ecrosswhite** [reply]
Thank you for the quick response, tested with `avd_data_validation_mode: warning` and it indeed worked where the CLI configuration was generated for the shutdown commands.

Thanks again

**Reply by ClausHolbechArista** [reply]
Fixed in #3100. The next release (4.3) will have this fix.

**Reply by ClausHolbechArista** [reply]
AVD 4.3 has been released on Friday, so please confirm that it solves this issue.
Thanks

**Reply by ecrosswhite** [reply]
I've confirmed that AVD 4.3 solves this issue, thank you!
